src/desktop/DesktopNativeCryptoFacade.ts

```diff
@@ -128,6 +134,8 @@ export class DesktopNativeCryptoFacade implements NativeCryptoFacade {
 		parallelism: number,
 		hashLength: number,
 	): Promise<Uint8Array> {
-		throw new Error("not implemented for this platform")
+		/** the fact that we don't want to define the cost parameters on four platforms makes it difficult to have the native
```

Comment:
I feel like we just don't need to use `WASMArgon2idFacade` and should use `generateKeyFromPassphraseArgon2id` directly. It is the right level of abstraction in this case.

We also wouldn't need a dance with file fetch and whatnot because streaming or not doesn't make such a huge difference to us.